src/cordova/setup.action.mjs

```diff
@@ -53,17 +53,19 @@ export async function main(...parameters) {
   if (!existsSync(path.resolve(process.env.ROOT_DIR, `platforms/${platform}`))) {
     await cordova.platform(
       'add',
-      [platform === 'osx' ? `github:apache/cordova-osx#${WORKING_CORDOVA_OSX_COMMIT}` : platform],
+      [platform === 'macos' ? `github:apache/cordova-osx#${WORKING_CORDOVA_OSX_COMMIT}` : platform],
       {save: false}
     );
   }
 
-  await cordova.prepare({platforms: [platform], save: false});
+  await cordova.prepare({platforms: [platform === 'macos' ? 'osx' : platform], save: false});
```

Comment:
Can you create a `cordova_platform` variable that is mapped from platform and you use in the cordova context?
Or maybe a function to map to cordova platform.